packages/web3-eth-accounts/src/common/types.ts

```diff
@@ -155,55 +148,44 @@ export type PrefixedHexString = string;
 /*
  * A type that represents an input that can be converted to a Buffer.
  */
-export type BufferLike =
-	| Buffer
-	| Uint8Array
-	| number[]
-	| number
-	| bigint
-	| TransformableToBuffer
-	| PrefixedHexString;
+export type Uint8ArrayLike = Buffer | Uint8Array | number[] | number | bigint | PrefixedHexString;
```

Comment:
It's okay for `Buffer` to be included here and on `Line 139`?

Reply:
Nope, there is no need for that as every Buffer is Uint8Array. This will cause this type to be "any" in env where Buffer global is not available^^